deeplabcut/pose_estimation_pytorch/modelzoo/inference.py

```diff
@@ -181,6 +187,9 @@ def _video_inference_superanimal(
             bbox=bbox,
             cmap=colormap,
             output_path=str(output_video),
+            plot_bboxes=plot_bboxes,
+            bboxes_list=bboxes_list,
+            bboxes_pcutoff=bboxes_pcutoff,
```

Comment:
Hey, can we do something about this method's import? This `_create_labeled_video()` is defined in another module and is supposed to be private, so not imported here... I haven't found what would be the best way to resolve this

Reply:
I agree that it's not great to have this. There are different ways to do this

- add `create_labeled_video` method which calls the private method
- rename the method and add `_create_labeled_video = create_labeled_video` for backwards compatibility (note: do we need to? a private method should not be used externally)
- call `CreateVideo` instead (not good; there would likely be code duplication)

Reply:
Agreed that this should be fixed. I think we could either:

- rename the method to `create_labeled_video` (as the method was private, do we need to add `_ create_labeled_video = create_labeled_video`? normally not, but I think it might be best to do it anyways)
- add a `create_labeled_video` wrapper around `_create_labeled_video` (not great)
- call `CreateVideo` instead (not great either; likely code duplication)

Reply:
To be sure I understand you right, the first option is to rename `_create_labeled_video()` to `make_video()` [for example] ?
And then you were wondering if we should add `_create_labeled_video` = `make_video` (that is what you meant, right)? I would say that it is not needed

Reply:
Yes that's what I meant - by not needed, do you mean because `_create_labeled_video` is private and thus nobody should be calling it from outside that file?

I agree that in principle that should be the case, however I think there's a chance other people were using it as it's a useful method that shouldn't have been made private, and adding this mapping would ensure outside scripts don't break

Reply:
Ok, yes that was my reasoning, but ok I will add the link then!